x-pack/plugins/enterprise_search/public/applications/workplace_search/views/oauth_authorize/constants.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const HTTP_REDIRECT_WARNING_MESSAGE = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.oauthAuthorize.httpRedirectWarningMessage',
+  {
+    defaultMessage: 'This application is using an insecure redirect URI (http)',
+  }
+);
+
+export const SCOPES_LEAD_IN_MESSAGE = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.oauthAuthorize.scopesLeadInMessage',
+  {
+    defaultMessage: 'This application will be able to',
+  }
+);
+
+export const AUTHORIZATION_REQUIRED_TITLE = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.oauthAuthorize.AuthorizationTitle',
+  {
+    defaultMessage: 'Authorization required',
+  }
+);
+
+export const AUTHORIZE_BUTTON_LABEL = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.oauthAuthorize.AuthorizeButtonLabel',
+  {
+    defaultMessage: 'Authorize',
+  }
+);
+
+export const DENY_BUTTON_LABEL = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.oauthAuthorize.DenyButtonLabel',
+  {
+    defaultMessage: 'Deny',
+  }
+);
+
+export const SEARCH_SCOPE_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.oauthAuthorize.SearchScopeDescription',
```

Comment:
Can we use camelCase here?
```suggestion
  'xpack.enterpriseSearch.workplaceSearch.oauthAuthorize.searchScopeDescription',
```

Reply:
whoops! I think I introduced that extra dot and missed downcasing the next word.